### PR TITLE
Tidying frontend VariableProvider tests

### PR DESCRIPTION
**Pull request by kkatzen**

### CODE REVIEW

File: frontend/src/data/variables/TestUtils.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+export const ASIAN = "Asian (Non-Hispanic)";
+export const TOTAL = "Total";
+export const RACE = "race_and_ethnicity";
+export const AGE = "age";
+export const SEX = "sex";
+
+export interface FipsSpec {
+  code: string;
+  name: string;
+}
```
Comment (aaronsn) [inline frontend/src/data/variables/TestUtils.ts:18]: Optional nit: potentially some of these could be in a common non-test file?

File: frontend/src/data/variables/TestUtils.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+export const MARIN: FipsSpec = {
+  code: "06041",
+  name: "Marin County",
+};
+export const USA: FipsSpec = {
+  code: USA_FIPS,
+  name: USA_DISPLAY_NAME,
+};
+
+export async function evaluateWithAndWithoutTotalInternal(
```
Comment (aaronsn) [inline frontend/src/data/variables/TestUtils.ts:44]: Optional nit: I usually don't start filename with a capital letter unless they export something with the same name. Just a personal preference.

Comment (aaronsn) [inline frontend/src/data/variables/TestUtils.ts:44]: I'd maybe recommend making this function `createWithAndWithoutTotalEvaluator` that returns a function

**Review by aaronsn — APPROVED** [review]

File: frontend/src/data/variables/TestUtils.ts (hunk 2 above)
Comment (kkatzen) [reply frontend/src/data/variables/TestUtils.ts:44]: oo neat idea

**Pull request merged**